src/proof/arith_proof_recorder.cpp

```diff
@@ -30,12 +30,19 @@ ArithProofRecorder::ArithProofRecorder() : d_lemmasToFarkasCoefficients()
 void ArithProofRecorder::saveFarkasCoefficients(
     Node conflict, theory::arith::RationalVectorCP farkasCoefficients)
 {
+  // Verify that the conflict is a conjuction of (possibly negated) real bounds
+  // Verify that the conflict is a conjunciton ...
   Assert(conflict.getKind() == kind::AND);
   Assert(conflict.getNumChildren() == farkasCoefficients->size());
   for (size_t i = 0; i < conflict.getNumChildren(); ++i)
```

Comment:
I know it's not part of this PR, but since you already touched the code around it, can you maybe change this loop header to not call `getNumChildren()` in each iteration?

Reply:
Sure (see above).